tests/rest/test_single_topology_rest.py

```diff
@@ -71,9 +78,11 @@ def test_single_topology_rest_vacuum(mol_pair, temperature_scale_interpolation_f
     st = get_single_topology(mol_a, mol_b, core)
     st_rest = get_single_topology_rest(mol_a, mol_b, core, 2.0, temperature_scale_interpolation_fxn)
 
+    assert 0 < len(st_rest.rest_region_atom_idxs) < st_rest.get_num_atoms()
```

Comment:
does this actually work? i.e. does it evaluate to:

(0 < len(st_rest.rest_region_atom_idxs) == {True,False}) < st_rest.get_atoms()?

Reply:
Admittedly seems a bit cursed but it does work as you say: https://docs.python.org/3/reference/expressions.html#comparisons

> Comparisons can be chained arbitrarily, e.g., x < y <= z is equivalent to x < y and y <= z, except that y is evaluated only once